types/react-dom/index.d.ts

```diff
@@ -12,12 +13,18 @@ export as namespace ReactDOM;
 import {
     ReactInstance, Component, ComponentState,
     ReactElement, SFCElement, CElement,
-    DOMAttributes, DOMElement
+    DOMAttributes, DOMElement, ReactNode, ReactPortal
 } from 'react';
 
 export function findDOMNode(instance: ReactInstance): Element;
 export function unmountComponentAtNode(container: Element): boolean;
 
+export function createPortal(
+    children: ReactNode,
+    container: Element,
+    implementation?: any,
```

Comment:
I think the third argument here should be `key`, per https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/fiber/ReactDOMFiberEntry.js#L735-L738